middlewares/isAdmin.js

```diff
@@ -0,0 +1,11 @@
+const { User } = require('../database/models')
+
+function isAdmin(req, res, next) {
+  const user = User.findById(req.roleId)
```

Comment:
aca deberia ser un req.user and req.user.roleId === (el id que le pongan al admin).  En este middleware no deberian ir de nuevo a buscar el usuario a la db

Reply:
Bueno, hardcodeado en 1 en lugar de ir a buscarlo a la tabla de roles 😛

Reply:
No, eso se iria del scope. Lo que si pueden hacer es tener el id como una variable de entorno, entonces ahi quedaria mas prolija y mas facil para setear si tienen un cambio. Sino, hay una forma de que el roleId vuelva con el tipo de usuario. Eso se hace con unas propiedades de sequelize... Pero no al complicaria tanto ahora. Si quieren lo dejamos para mas adelante

Reply:
Bien pensada la solución! 
Habitualmente perfil/rol administrador per sé hay uno solo, pero sí puede pasar que a otro usuario no admin se le de permisos para modificar cierta función. Ahí de nuevo entramos en debate de scope de tarjeta vs escalabilidad de la aplicación...quizás para un middleware de validación de si es puntualmente admin no justifica esto, pero tranquilamente puede evolucionar a middleware de permisos luego como lo hiciste ahora.